plugins/source/postgresql/client/sync.go

```diff
@@ -108,7 +108,7 @@ func (c *Client) syncTables(ctx context.Context, snapshotName string, filteredTa
 }
 
 func syncTable(ctx context.Context, tx pgx.Tx, table *schema.Table, res chan<- message.SyncMessage) error {
-	colNames := make([]string, 0, len(table.Columns)-2)
```

Comment:
Where do you think this came from? internal columns?

Reply:
Not sure, maybe 🤷 Maybe CoPilot?